xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererMediaCodec.cpp

```diff
@@ -116,8 +116,8 @@ CRenderInfo CRendererMediaCodec::GetRenderInfo()
 {
   CRenderInfo info;
   info.formats = m_formats;
-  info.max_buffer_size = NUM_BUFFERS;
-  info.optimal_buffer_size = 2;
+  info.max_buffer_size = 4;
```

Comment:
https://github.com/xbmc/xbmc/pull/9542#issuecomment-207771279

I remember that skipping was not smooth anymore and a/v desync was to high. 

> Or do you need at least fps/Hz*2 buffers? 

I don't think so. iirc I tested with TV set to 24/23.975 Hz.